Meta/run.py

```diff
@@ -484,7 +484,7 @@ def set_up_disk_image_path(config: Configuration):
 
 def set_up_cpu(config: Configuration):
     if config.qemu_kind == QEMUKind.NativeWindows:
-        config.qemu_cpu = "max,vmx=off"
+        config.qemu_cpu = None
```

Comment:
nit: Maybe add a comment that `-cpu max` breaks on QEMU 9 with WHPX

Reply:
Sure, I've added a comment above that line